commons/src/main/java/org/infinispan/commons/util/concurrent/jdk8backported/BoundedEquivalentConcurrentHashMapV8.java

```diff
@@ -2377,8 +2383,8 @@ public BoundedEquivalentConcurrentHashMapV8(long maxSize, int initialCapacity,
          throw new IllegalArgumentException();
       if (initialCapacity < concurrencyLevel)   // Use at least as many bins
          initialCapacity = concurrencyLevel;   // as estimated threads
-      long size = (long)(1.0 + (long)initialCapacity / loadFactor);
-      int cap = (size >= (long)MAXIMUM_CAPACITY) ?
+      long size = (long)(1.0 + initialCapacity / loadFactor);
+      int cap = (size >= MAXIMUM_CAPACITY) ?
```

Comment:
Probably not, tbh I just ignored the changes in this file since they seemed completely unnecessary but fine.